AK/Vector.h

```diff
@@ -314,15 +321,16 @@ requires(!IsRvalueReference<T>) class Vector {
         MUST(try_prepend(values, count));
     }
 
-    // FIXME: What about assigning from a vector with lower inline capacity?
-    Vector& operator=(Vector&& other)
+    template<class U, size_t inline_capacity_u>
+    requires CanMoveIn<inline_capacity, inline_capacity_u>
```

Comment:
...you can just plop the definition here instead:
```suggestion
    requires(inline_capacity >= inline_capacity_u)
```